gulpfile.js

```diff
@@ -1141,6 +1142,13 @@ gulp.task("jsdoc", function (done) {
   });
 });
 
+gulp.task("dts", function (done) {
```

Comment:
`dts` doesn't feel like a very descriptive task name, can we call it e.g. `types` instead?

Basically, let's replace all `dts`/`DTS` occurrences with `types`/`TYPES` instead.

Reply:
done